hvac/api/secrets_engines/identity.py

```diff
@@ -335,7 +335,12 @@ def list_entities_by_name(self, method="LIST", mount_point=DEFAULT_MOUNT_POINT):
         return response
 
     def merge_entities(
-        self, from_entity_ids, to_entity_id, force=None, mount_point=DEFAULT_MOUNT_POINT
+        self, 
+        from_entity_ids, 
+        to_entity_id, 
+        force=None, 
+        conflicting_alias_ids_to_keep=None,
+        mount_point=DEFAULT_MOUNT_POINT
```

Comment:
we should swap the order of these so that we don't break anyone using these arguments positionally
```suggestion
        mount_point=DEFAULT_MOUNT_POINT,
        conflicting_alias_ids_to_keep=None,
```